rdkafka-sys/Cargo.toml

```diff
@@ -10,7 +10,7 @@ description = "Native bindings to the librdkafka library"
 keywords = ["kafka", "rdkafka"]
 
 [dependencies]
-# lz4-sys = { git = "https://github.com/thijsc/lz4-rs.git" }
+lz4-sys = ">= 1.0"
 libz-sys = ">= 1.0.7"
 openssl-sys = "> 0.7.0"
```

Comment:
Part of the inequality is missing (we don't want major version changes). We should probably use `^`, `~` or `*` instead.